test/qa.bats

```diff
@@ -50,9 +50,9 @@ EOF
     rm -rf "${tmpDir}"
 }
 
-@test "ensure we have nodejs 18 installed" {
+@test "ensure we have nodejs installed" {
     run docker run --rm --entrypoint=node "${TEST_IMAGE}" --version
-    [[ "${output}" =~ v18\.[0-9]+\.[0-9]+ ]]
+    [[ "${output}" =~ [0-9]+\.[0-9]+\.[0-9]+ ]]
```

Comment:
Why not test if it is >=18?

Reply:
Valid point thanks, Istvan! We will do a hardening on the tests for now this is verifying that a node version is installed.